packages/entity-server/src/routes/hierarchy/EntityDescendantsRoute.ts

```diff
@@ -13,23 +13,24 @@ import {
   EntityResponseObject,
 } from './types';
 
+export type DescendantsRequest = HierarchyRequest<
+  HierarchyRequestParams,
+  EntityResponseObject[],
+  HierarchyRequestBody,
+  HierarchyRequestQuery & { includeRootEntity?: boolean }
+>;
 export class EntityDescendantsRoute extends Route<
-  HierarchyRequest<
-    HierarchyRequestParams,
-    EntityResponseObject[],
-    HierarchyRequestBody,
-    HierarchyRequestQuery & { includeRootEntity?: boolean }
-  >,
+  DescendantsRequest,
   HierarchyResponse<EntityResponseObject[]>
 > {
   async buildResponse() {
     const { includeRootEntity = false } = this.req.query;
-    const descendants = await this.req.context.entity.getDescendants(this.req.context.hierarchyId, {
-      country_code: this.req.context.allowedCountries,
+    const descendants = await this.req.ctx.entity.getDescendants(this.req.ctx.hierarchyId, {
```

Comment:
I blanket renamed `context` to `ctx` cos I've seen it around and it's quicker to write 😋 Does it read easy to you?

Reply:
They are fine either way! With short names, I've found that it's largely a matter of getting used to them (although they become hard to read when everything is shortened)